hoot-core/src/main/cpp/hoot/core/visitors/IndexElementsVisitor.cpp

```diff
@@ -47,12 +47,12 @@ namespace hoot
 IndexElementsVisitor::IndexElementsVisitor(boost::shared_ptr<HilbertRTree>& index,
                                            deque<ElementId>& indexToEid,
                                            const boost::shared_ptr<ElementCriterion>& criterion,
-                                           boost::function<Meters (const ConstElementPtr& e)> getSearchRadius,
+                                 boost::function<Meters (const ConstElementPtr& e)> getSearchRadius,
```

Comment:
Of all of the reformatting that makes everything evened up, this one went the wrong way.